README.md

```diff
@@ -119,6 +119,10 @@ To do so:
 * Replace the logo image `logo.png` with `logo.svg` in your templates
 * Update anything app-specific that needs updating - the whole masthead region should have a white background
 
+## Generating favicons
+
+If you need to update the favicons, you can amend the settings (or add new code) to `generate-icons.js` and recreate the icons by running `npm run generateIcons`.
```

Comment:
We will probably need to build on this for someone consuming ID7 for their app, and wouldn't want to be modifying the id7 library. We'd allow passing a custom directory of source images plus options like background colours, and let it build from that into a chosen output directory. But can work on that separately.